lib/Demangling/Remangler.cpp

```diff
@@ -3327,6 +3327,7 @@ ManglingError Remangler::mangleOpaqueType(Node *node, unsigned depth) {
   if (trySubstitution(node, entry))
     return ManglingError::Success;
 
+  DEMANGLER_ASSERT(node->getNumChildren() >= 2, node);
```

Comment:
Fred pointed out this should be `3` not two (below `node->getChild(2)` is accessed).